controls/class-settings-page-text-input-control.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+class Settings_Page_Text_Input_Control extends Field_Control {
+
+	/**
+	 * @var mixed|string
+	 */
+	protected $section;
+	/**
+	 * @var string
+	 */
+	protected $description;
+	/**
+	 * @var string
+	 */
+	protected $page;
+
+	public function __construct( string $id, string $control_id, array $args = [] ) {
+		parent::__construct( $id, $control_id, $args );
+		$this->section     = (string) $args['section'] ? : '';
+	}
+
+	public function get_content() : string {
+		return "<input type='text' name='" . esc_attr( $this->get_id() ) . "' id='" . esc_attr( $this->get_id() ) . "' value='" . esc_attr( $this->get_datastore()->get_value() ) . "'>";
```

Comment:
It's important to any accessibility-first modeling that we ensure that fields always include labels. We should make sure to structure classes to require a label and ensure it is an inherent part of the output.

Reply:
Are there cases when the label would not be rendered in direct proximity to the `<input>`? 

If so, it would be more flexible to create a `get_label()` method and output it inside `render()`.

Reply:
The label should always be rendered immediately before or after the input (after is standard for checkboxes and radio inputs), but if we want to support a mechanism for people to customize HTML wrappers, then I think having a separate method provides more flexibility.

Reply:
Current settings screen uses tables with labels on the left and inputs on the right in separate table cells. That's the use case we have to account for here -- at least currently -- since the usage of these controls will vary widely depending on implementation.

Reply:
I propose `get_content()` returns the label + input, and we have separate methods for the label/input.

Reply:
Or we use `render()` for that combined purpose and remove `get_content()`